account-keys/slip10/src/mnemonic2slip.py

```diff
@@ -48,28 +55,68 @@ def seed2hdnode(seed, modifier, curve):
         if a < curve.order and a != 0:
             break
         seed = h
-        # print 'RETRY seed: ' + binascii.hexlify(seed)
+    return key, chaincode
+
+
+def publickey(private_key, curve):
+    if curve == 'ed25519':
+        sk = ed25519.SigningKey(private_key)
+        return b'\x00' + sk.get_verifying_key().to_bytes()
+    else:
+        Q = string_to_int(private_key) * curve.generator
+        xstr = int_to_string(Q.x(), 32)
+        parity = Q.y() & 1
+        return chr(2 + parity) + xstr
+
+
+def derive(parent_key, parent_chaincode, i, curve):
+    assert len(parent_key) == 32
+    assert len(parent_chaincode) == 32
+    k = parent_chaincode
```

Comment:
weird unused `k` again - is there maybe a deeper meaning here related to python malleability?